cmd/api/src/daemons/datapipe/agi.go

```diff
@@ -154,20 +155,26 @@ func ParallelTagAzureTierZero(ctx context.Context, db graph.Database) error {
 	return nil
 }
 
-func TagActiveDirectoryTierZero(ctx context.Context, db graph.Database) error {
+func TagActiveDirectoryTierZero(ctx context.Context, db database.Database, graphDB graph.Database) error {
 	defer log.Measure(log.LevelInfo, "Finished tagging Active Directory Tier Zero")()
 
-	if domains, err := adAnalysis.FetchAllDomains(ctx, db); err != nil {
+	autoTagT0ParentObjectsFlag, err := db.GetFlagByKey(ctx, appcfg.FeatureAutoTagT0ParentObjects)
+	if err != nil {
+		log.Errorf("error getting AutoTagT0ParentObjects feature flag: %w", err)
+		return err
+	}
+
+	if domains, err := adAnalysis.FetchAllDomains(ctx, graphDB); err != nil {
```

Comment:
```suggestion
	if autoTagT0ParentObjectsFlag, err := db.GetFlagByKey(ctx, appcfg.FeatureAutoTagT0ParentObjects); err != nil {
		return err
	} else if domains, err := adAnalysis.FetchAllDomains(ctx, graphDB); err != nil {
```

Suggestion for maintaining consistency in inlining initializer statements as well as handling error logging up the stack.